docs/handbook/image-file-formats.rst

```diff
@@ -298,6 +298,9 @@ The :py:meth:`~PIL.Image.Image.open` method may set the following
 **exif**
     Raw EXIF data from the image.
 
+**comment**
+    A comment about the image.
```

Comment:
Should we have a "New in version 7.1.0."?

Reply:
Good call. Done